packages/jobs/lib/processor/processor.ts

```diff
@@ -21,10 +21,17 @@ export class Processor {
     start() {
         logger.info('Starting task processors');
         try {
+            const syncWorker = new ProcessorWorker({
+                orchestratorUrl: this.orchestratorServiceUrl,
+                groupKey: 'sync',
+                maxConcurrency: 200
+            });
+            syncWorker.start();
+
             const actionWorker = new ProcessorWorker({
                 orchestratorUrl: this.orchestratorServiceUrl,
                 groupKey: 'action',
-                maxConcurrency: 500
+                maxConcurrency: 200
```

Comment:
in temporal syncs and actions shares the same queue with a max concurrency at 500.
Using different queues in the orchestrator with an arbitrary max concurrency at 200. (we usually never have more than 2 digits running workflow in temporal)